backends-clickhouse/src/main/scala/org/apache/spark/sql/execution/datasources/v1/CHFormatWriterInjects.scala

```diff
@@ -69,7 +69,7 @@ trait CHFormatWriterInjects extends GlutenFormatWriterInjectsBase {
       sparkSession: SparkSession,
       options: Map[String, String],
       files: Seq[FileStatus]): Option[StructType] = {
-    throw new UnsupportedOperationException("CHFormatWriterInjects does not support inferSchema")
+    OrcUtils.inferSchema(sparkSession, files, options)
```

Comment:
@zzcclp this will change the behavior on ORC

Reply:
It actually doesn't change CH ORC's behavior

See code change in

https://github.com/oap-project/gluten/pull/3363/files/aaed3b9cbc9f163112939f0d114d5f592233794d#diff-e8f3f4105f5be5620cacd38aa5d1deef02ce9e45f12dbde1f92699fb55d769c2L92-L93

Reply:
@taiyang-li please check this

Reply:
it is ok